### PR TITLE
Resume interrupted updates at boot

### PR DESCRIPTION
**Pull request by radujipa**
This PR was to ensure that the `kano-updater ui boot-window` was up to date and capable of continuing an interrupted install. Some indirect improvements where made along with that. Changes are organised in commits to help with reviewing.

### Removed install confirm window at bootup

This simply removes the user confirmation window when updates have
been scheduled. Previously at shutdown, if there was an update
available the user would be asked if they wanted to install it
then or later. Subsequently, they would get asked again at bootup.
Now, the Dashboard will constantly show a message about installing
and this popup is no longer useful.

### Remove broken and unused post-update bootup changes dialog

It seems like there was an old mechanism of showing what was
installed after an update with an HTML page and a WebApp dialog.
When this was manually triggered, the Updater exited with a
traceback. Removing this part of the code along with any other

### Improve app launch time by eliminating D-Bus iface retries

This sets the retry_count=0 in other methods that were missed in https://github.com/KanoComputing/kano-updater/commit/10c1b9923a6d08853e36dac14fbcbdd699c79d29

### Fix binary executable permissions

For some reason these two binaries did not have executable
permissions??? **How did this ever work?**

### Others

 - Removed unused app assets (pngs, gifs)
 - Code styling and removing imports of unused modules

### CODE REVIEW

**Comment by radujipa** [comment]
[(5) As a user who unplugs during an upgrade, I want the updater to relaunch itself to finish](https://trello.com/c/IE04dtnM/1766-5-as-a-user-who-unplugs-during-an-upgrade-i-want-the-updater-to-relaunch-itself-to-finish)

File: debian/changelog [hunk 1]
```diff
@@ -4,6 +4,8 @@ kano-updater (3.15.0-0) unstable; urgency=low
   * Improve app launch time by eliminating D-Bus iface retries
   * Fixed Python package to be importable and not execute code at import
   * Download & Install commands now re-check and re-download over an old cache
+  * Removed deprecated ChangesDialog and app assets
+  * Fixed binary execution permissions
 
  -- Team Kano <dev@kano.me>  Thu, 25 Jan 2018 17:28:00 +0100
```
Comment (skarbat) [inline debian/changelog]: Date?

**Comment by radujipa** [comment]
This PR is now ready 🎉

File: conf/dhcpcd-hooks/70-kano-updater-download [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# 70-kano-updater-download
```
Comment (Ealdwulf) [inline conf/dhcpcd-hooks/70-kano-updater-download]: Was wondering if you needed a `#!` - it turns out `dhcpcd-run-hooks`  _sources_ these scripts; yuck.

File: conf/dhcpcd-hooks/70-kano-updater-download [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        min_uptime_sec=180
+        delta_uptime=$((min_uptime_sec-uptime_sec))
+
+        # Ensure that the background download only happens after min_uptime_sec
+        # This prevents multiple instances clashing during boot.
+        # TODO: This could've been a queued systemd service that only ran after
+        # kano-dashboard.service
+        if [ "$delta_uptime" -gt 0 ]; then
+
+            (sleep "$delta_uptime" && \
```
Comment (Ealdwulf) [inline conf/dhcpcd-hooks/70-kano-updater-download]: I don't think you actually need a `\` here. The shell knows you didn't terminate your `(` yet.

Comment (radujipa) [reply conf/dhcpcd-hooks/70-kano-updater-download]: Nice! Fixed

**Pull request merged**